extensions/binding/org.eclipse.smarthome.binding.sonos/README.md

```diff
@@ -22,7 +22,12 @@ The Sonos devices are discovered through UPnP in the local network and all devic
 
 ## Binding Configuration
 
-The binding does not require any special configuration
+The binding has the following configuration options, which can be set for "binding:sonos":
+
+| Parameter | Name    | Description  | Required |
+|-----------------|------------------------|--------------|------------ |
+| opmlUrl | OPML Service URL | URL for the OPML/tunein.com service | no |
+| callbackUrl | Callback URL | URL to use for playing notification sounds, e.g. http://192.168.0.2:8080 | no |
```

Comment:
This callbackURL specifies the url where the runtime can be reached, not the speaker.
It is mainly required if you have multiple interfaces in your host and thus your external IP cannot be easily determined (what is tried if you leave this value empty). In 99% of all setups, I would claim that all Sonos speakers should be fine with the same url for reaching the runtime.